Source/Core/DolphinNoGUI/MainNoGUI.cpp

```diff
@@ -280,22 +280,6 @@ int main(int argc, char* argv[])
   if (options.is_set("user"))
     user_directory = static_cast<const char*>(options.get("user"));
 
-  UICommon::SetUserDirectory(user_directory);
```

Comment:
Is this no longer needer?

Reply:
We still call it down below. I'm not sure how we ended up with the duplicate calls.